packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -61,7 +62,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         { ...{ onBlurCapture: this._onTooltipMouseLeave } }
         onMouseEnter={ this._onTooltipMouseEnter }
         onMouseLeave={ this._onTooltipMouseLeave }
-        aria-describedby={ isTooltipVisible ? tooltipId : undefined }
+        aria-describedby={ (isTooltipVisible && !stopAriaDescribedBy) ? tooltipId : undefined }
```

Comment:
@petdun2519 I mean if the overflowmode is set to anything, it's probably a pretty good indicator that the tooltip is meant to show text that would otherwise be visually hidden. In that case, we could not set the aria-describedby?